src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  curentKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    curentKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ curentKey: event.key });
```

Comment:
Do not duplicate code(here and lines 19-20). Create a separate function and use it.